addons/udes_stock/report/deliveryslip_templates.xml

```diff
@@ -7,147 +7,139 @@
+                <t t-set="o" t-value="o.with_context({'lang':o.partner_id.lang})"/>
 
-                    <t t-if="o.state != 'done'">
-                        <h1>
-                            DRAFT
-                        </h1>
-                    </t>
+                <t t-if="o.state != 'done'">
+                    <h1>
+                        DRAFT
+                    </h1>
+                </t>
 
-                    <h2>
-                        Delivery Slip
-                    </h2>
+                <h2>
+                    Delivery Slip
+                </h2>
 
-                    <!-- To avoid to break sale_stock -->
-                    <div name="customer_address" />
-                    <div name="div_sched_date" />
+                <!-- To avoid to break sale_stock -->
+                <div name="customer_address"/>
+                <div name="div_sched_date"/>
 
-                    <table class="table table-condensed">
-                        <thead>
-                            <tr>
-                                <th><strong>Trailer number</strong></th>
-                                <th><strong>Trailer unit ID</strong></th>
-                                <th><strong>Vehicle registration</strong></th>
-                                <th><strong>Driver name</strong></th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            <tr>
-                                <td>
-                                    <span t-field="o.u_vehicle_sequence"/>
-                                </td>
-                                <td>
-                                    <span t-field="o.u_vehicle_description"/>
-                                </td>
-                                <td>
-                                    <span t-field="o.u_vehicle_registration"/>
-                                </td>
-                                <td>
-                                    <span t-field="o.u_driver_name"/>
-                                </td>
-                            </tr>
-
-                            <t t-if="o.partner_id">
-                                <tr>
-                                    <th colspan="2" style="padding-top: 40px" />
-                                    <th colspan="2" style="padding-top: 40px"><strong>Customer address</strong></th>
-                                </tr>
-                                <tr>
-                                    <td colspan="2" />
-                                    <td colspan="2">
-                                        <span t-field="o.partner_id"
-                                           t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True}'/>
-                                    </td>
-                                </tr>
-                            </t>
```

Comment:
This has been removed after a discussion with PO. It is not needed in delivery slips.